tests/post_training/data/wc_reference_data.yaml

```diff
@@ -1,22 +1,22 @@
 tinyllama_data_free_backend_OV:
   metric_value: 0.72057
-  num_int4: 228
+  num_int4: 114
```

Comment:
Why is it twice lower? I suppose, the counting of int4 layer should be modified as well

Reply:
Because in the symmetric case, we don't have zero points anymore.

Reply:
I see, they were counted with weights as well.